examples/common/common.rs

```diff
@@ -43,13 +43,15 @@ pub fn post<T: Serialize + ?Sized>(json: &T) -> Result<Response> {
     Ok(response)
 }
 
+#[allow(dead_code)]
```

Comment:
this is needed because file is imported in 2 examples, and one of them is not using some functions.

This was the only way I saw to share code between examples without creating a new crate.

Reply:
sadly this is also the only way I know of.